src/SWP/Bundle/CoreBundle/Resources/config/serializer/Model.ContentList.yml

```diff
@@ -1,5 +1,17 @@
 SWP\Bundle\CoreBundle\Model\ContentList:
     exclusion_policy: ALL
+    properties:
```

Comment:
I didnt - I dont see it in 0.1 branch: https://github.com/superdesk/web-publisher/blob/0.1/src/SWP/Bundle/CoreBundle/Resources/config/serializer/Model.ContentList.yml